core/src/banking_stage/transaction_scheduler/transaction_state.rs

```diff
@@ -294,16 +294,16 @@ mod tests {
     }
 
     #[test]
-    fn test_transaction_priority_details() {
+    fn test_compute_budget_details() {
         let priority = 15;
```

Comment:
nit: rename to `compute_unit_price()`.

Reply:
5fd4a33bfa357e4ebf037eeacda5c57a1729306e